### DISCUSSION TITLE
求助，电脑版无法通过授权设备，也无法通过邮箱码授权登录专业版

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#860] [Q&A]
@Mike777Khoo 你好，已购买专业版，账号：firekydchan@gmail.com

电脑端：一直显示为免费版状态，无法通过授权设备，也无法通过邮箱码授权登录专业版，多次登录，重启程序、重装均无效；
手机端：可正常登录，但无法授权电脑端，绑定设备一直只能显示一台

![微信截图_20240911205408](https://github.com/user-attachments/assets/708e6cdd-16ff-42b9-9bbb-709e5d8cb54f)

错误代码：request failed with status code 503

版本：7.9.5 (20240823.185949)

ticket号码：76844694，logs ：
[lantern.log](https://github.com/user-attachments/files/16981907/lantern.log)

希望能帮忙解决，感谢

**Comment by MaxMKMax** [comment]
@kidokogills 谢谢您的log. 我们会重视您的反馈. 尽快帮您修复问题. 但是要等到下周二我会和技术部门开会.
